stable/atlantis/templates/statefulset.yaml

```diff
@@ -72,6 +72,9 @@ spec:
           {{- if .Values.allowRepoConfig }}
             - --allow-repo-config
           {{- end }}
+          {{- if .Values.silenceWhitelistErrors }}
+            - --silence-whitelist-errors
```

Comment:
Use the environment variable instead please. This will match the rest of the chart